css/css-align/self-alignment/block-justify-self.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Self-Alignment: justify-self - block-level elements</title>
+<link rel="author" title="David Tran" href="mailto:davidtranhq+wpt@gmail.com">
+<link rel="match" href="block-justify-self-ref.html">
+<link rel="help" href="https://www.w3.org/TR/css-align-3/#justify-block" />
+<meta name="assert" content="justify-self applies to block-level boxes">
+<style>
+.grid-container div {
+    width: 100px;
+    background-color: green;
+}
+
+.block-container div, .block-container-with-table-items div {
+    width: 100px;
+    background-color: red;
+}
+
+
+.stack {
+    display: grid;
+}
+
+.stack > * {
+    grid-row: 1;
+    grid-column: 1;
+}
+
+</style>
+</head>
+<p>Test passes if <strong>no red</strong> is visible</p>
+
+<div class="stack">
+<article class="block-container" style="display: block; width: 300px">
+  <div style="justify-self: auto;">auto</div>
+  <div style="justify-self: left;">left</div>
+  <div style="justify-self: right;">right</div>
+  <div style="justify-self: start;">start</div>
+  <div style="justify-self: center;">center</div>
+  <div style="justify-self: end;">end</div>
+  <div style="justify-self: self-start;">self-start</div>
+  <div style="justify-self: self-end;">self-end</div>
+  <div style="justify-self: flex-start;">flex-start</div>
+  <div style="justify-self: flex-end;">flex-end</div>
+  <div style="display: float; justify-self: center;">float</div>
```

Comment:
This should be `float: left` / `float: right`; there's no such thing as `display: float`.